src/vma/dev/rfs_uc_tcp_gro.cpp

```diff
@@ -47,7 +49,15 @@ rfs_uc_tcp_gro::rfs_uc_tcp_gro(flow_tuple *flow_spec_5t, ring_simple *p_ring, rf
 	m_p_gro_mgr(&(p_ring->m_gro_mgr)), m_b_active(false), m_b_reserved(false)
 {
 	m_n_buf_max = m_p_gro_mgr->get_buf_max();
-	m_n_byte_max = m_p_gro_mgr->get_byte_max() - p_ring->get_mtu(); 
+	int mtu = p_ring->get_mtu();
+	route_result res;
+	// m_tos is always 0 in VMA
+	g_p_route_table_mgr->route_resolve(route_rule_table_key(flow_spec_5t->get_dst_ip(),
+					flow_spec_5t->get_src_ip(), 0), res);
+	if (res.mtu) {
+		mtu = res.mtu;
+	}
```

Comment:
do we need `if (res.mtu && (res.mtu < mtu))` in this place?
do we need to use a minimum MTU value for route and device?